org.gnu.emacs.json

```diff
@@ -155,17 +155,12 @@
             "sources": [
                 {
 		    "type": "git",
-                    "url": "https://github.com/emacs-mirror/emacs.git",
-		    "branch": "feature/pgtk"
+                    "url": "https://github.com/emacs-mirror/emacs.git"
                 },
                 {
                     "type": "patch",
                     "path": "appdata-launchable.patch"
                 },
-                {
-                    "type": "patch",
-                    "path": "appdata-id.patch"
```

Comment:
I missed uploading this patch the first time. (I have fixed it now)

I'd like to keep it as it effectively separates this package from the flathub one, as this is not a default compile option